src/pug/blog/year_one_recap.pug

```diff
@@ -0,0 +1,67 @@
+extends _blog.pug
+
+block append subtitle
+  | Year one recap
+
+block content
+  h2 Year one recap
+  +news_header('22 November 2023', 'report_18', '')
+
+  p One year ago Input Labs went live, it is time to review our achievements and our failures.
+
+  h3 Things that went nicely
+  h4 Community
+  ul
+    li Our nice #[em community kept growing], now with 735 discord members. This is by far the thing that makes us more proud, and it is awesome to see people helping each other there. To make it even better, some of the members are experts in their fields (electrical engineers, industrial designers) and contribute with expert advice and knowledge.
+    li We got a #[em small office] in the #[+ax('https://maria.io/about/') Maria 01 startup campus] in Helsinki, and we moved our 3D-printers and other tools there, making it the "Lab" of Input Labs.
+    li The project received plenty of #[em positive feedback] in the different social channels in which we have presence, and lot of people encouraged us to continue. We got very limited (almost negligible) negative interactions.
```

Comment:
Rephrase the last sentence. Something like "poor communication or  inappropriate interaction was pretty much negligible"